src/engraving/dom/guitarbend.cpp

```diff
@@ -0,0 +1,502 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "accidental.h"
+#include "chord.h"
+#include "editdata.h"
+#include "guitarbend.h"
+#include "note.h"
+#include "score.h"
+#include "tie.h"
+
+namespace mu::engraving {
+/****************************************
+ *              GuitarBend
+ * **************************************/
+
+GuitarBend::GuitarBend(EngravingItem* parent)
+    : SLine(ElementType::GUITAR_BEND, parent, ElementFlag::MOVABLE)
+{
+}
+
+GuitarBend::GuitarBend(const GuitarBend& g)
+    : SLine(g)
+{
+    _type = g.type();
+    if (g.m_holdLine) {
+        m_holdLine = new GuitarBendHold(*g.m_holdLine);

```

Comment:
Shouldn't it be marked as "generated" (assuming this class deletes it explicitly)?

Reply:
Actually I just realized that it is completely unnecessary to copy it: it gets recalculated during layout anyway. About "generated": I don't think that flag is ever used for spanners, but anyway it doesn't hurt to set it